docs/plans/2026-05-13-player-wiki-and-help-redesign-design.md

```
@@ -0,0 +1,385 @@
+# LumaGuilds Player Wiki + `/g help` Redesign — Design
+
+**Date:** 2026-05-13
+**Status:** Draft for review
+**Author:** Brainstorm session (Noah + Claude)
+
+---
+
+## 1. Goals & Non-Goals
+
+### Goals
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)